### ISSUE TITLE
Antivirus test set updates

### ISSUE THREAD

**Issue by paulprokhorovmoz** [mozilla/iris2#401]
Please update appropriate tests according to updates:  
https://testrail.stage.mozaws.net/index.php?/cases/view/217869
https://testrail.stage.mozaws.net/index.php?/cases/view/225144
https://testrail.stage.mozaws.net/index.php?/cases/view/219580
https://testrail.stage.mozaws.net/index.php?/cases/view/225143

**Comment by cfogelSV** [comment]
So that the information doesn't get lost, here are the updates from the QA_manual team:

(1) https://testrail.stage.mozaws.net/index.php?/cases/view/217869
Updated preconditions:
- we've added clearer steps for Avast(how to enable that webshield);
- for other antiviruses, the alternative would now be to enable all the available web/browsing shields to the max_protection level.

(2) https://testrail.stage.mozaws.net/index.php?/cases/view/225144
@ step3: updated the page from about:preferences to about:preferences#privacy so it can be accessed faster if copied via link.

(3) https://testrail.stage.mozaws.net/index.php?/cases/view/219580
- removal of current step 5 (Click on the Trigger update button for the google provider.)
button removed from section.

(4) https://testrail.stage.mozaws.net/index.php?/cases/view/225143
- updating preconditions to mark as not applicable if preff is locked by the antivirus install
* consider bumping down the priority for this test case.
